src/styles/index.scss

```diff
@@ -1,3 +1,143 @@
+:root {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  box-sizing: border-box;
+}
+
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$button-color: #00acdc;
+$text-color: #fff;
+
 body {
   margin: 0;
 }
+
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  &__img {
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+  }
+
+  &__about {
+    display: flex;
+    flex-direction: column;
+    margin: 0 16px 16px;
+  }
+
+  &__title {
+    color: $main-accent-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    padding: 0;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rate {
+    margin-left: 16px;
+    margin-right: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price {
+    margin-left: 17px;
+    margin-right: 17px;
+    padding: 0;
+    margin-top: 6px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__buy:link {
+    text-decoration: none;
+    text-transform: uppercase;
+    border: none;
+    color: $text-color;
+    font-weight: 700;
+    line-height: 16px;
+    font-size: 14px;
+    border-radius: 5px;
+    background: $button-color;
+    padding: 12px 70px 12px 69px;
+    margin-left: 16px;
+    margin-right: 16px;
+  }
+
+  &__buy:visited {
+    color: $text-color;
+  }
+
+  &__buy:hover {
+    padding-top: 11px;
+    padding-bottom: 11px;
+    box-sizing: border-box;
+    border: 1px solid $button-color;
+    background: $text-color;
+    color: $button-color;
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 0;
+  margin-left: 2px;
+}
+
+.stars__star {
+  margin-top: 2px;
+  margin-right: 8px;
+  box-sizing: border-box;
+  width: 12px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-image: url(../images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.description {
+  color: $main-accent-color;
+  text-align: right;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.title {
+  color: $secondary-color;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.value {
+  color: $main-accent-color;
```

Comment:
Looks like these classes are "card" elements, let's move them to the "card" BEM block.

Checklist:
6. [BEM] - Don't make BEM structure too complicated. You don't need to create separate blocks for anything other than card itself and star